src/include/mujoco.cmake

```diff
@@ -1,11 +1,17 @@
+if(CMAKE_SYSTEM_NAME MATCHES "Linux")
+  set(MUJOCO_PATH /opt/mujoco-3.2.6-linux-x86_64/mujoco-3.2.6/)
+elseif (CMAKE_SYSTEM_NAME MATCHES "Darwin")
+  set(MUJOCO_PATH /Application/MuJoCo.app/mujoco.framework)
+endif()
+
 # For Mujoco
-list(APPEND CMAKE_PREFIX_PATH "/usr/local/")
-find_package(mujoco REQUIRED)
-target_include_directories(${LF_MAIN_TARGET} PUBLIC /usr/local/include)
-target_link_libraries(${LF_MAIN_TARGET} PUBLIC mujoco::mujoco)
+list(APPEND CMAKE_PREFIX_PATH ${MUJOCO_PATH})
+find_library(MUJOCO_LIB mujoco REQUIRED)
+find_path(MUJOCO_INCLUDE_PATH mujoco REQUIRED)
+target_include_directories(${LF_MAIN_TARGET} PUBLIC ${MUJOCO_INCLUDE_PATH})
+target_link_libraries(${LF_MAIN_TARGET} PUBLIC ${MUJOCO_LIB})
 
 # For GLFW
-list(APPEND CMAKE_PREFIX_PATH "/opt/homebrew/")
-find_package(glfw3 3.4 REQUIRED)
-target_include_directories(${LF_MAIN_TARGET} PUBLIC /opt/homebrew/include)
+find_package(glfw3 REQUIRED)
+target_include_directories(${LF_MAIN_TARGET} PUBLIC ${glfw3_INCLUDE_DIRS})
```

Comment:
Does this work on macOS? Is a little unfortunate to have that `/opt/homebrew/include` path here

Reply:
I have verified that this works